src/routes/reference/basic-reactivity/create-signal.mdx

```diff
@@ -3,7 +3,7 @@ title: createSignal
 ---
 
 Signals are the most basic reactive primitive. 
-They track a single value (which can be any JavaScript object) that changes over time.
+They track a single value (which can be a value of any type) that changes over time.
```

Comment:
I'm not a huge fan of this update; it feels imprecise. I think I'd be fine with `any JavaScript value`, but it seems useful to reference the language here.